apps/torus-worker/src/index.ts

```diff
@@ -16,21 +17,45 @@ export const env = parseEnvOrExit(
   }),
 )(process.env);
 
+const log = createLogger({ name: "index" });
```

Comment:
`index` means nothing. this should be at least "torus-web-worker" or something like that.